src/MasterSite.php

```diff
@@ -732,9 +735,32 @@ function ($item) {
         // Footer context.
         $context['copyright_text_line1'] = $options['copyright_line1'] ?? '';
         $context['copyright_text_line2'] = $options['copyright_line2'] ?? '';
-        $context['footer_social_menu'] = wp_get_nav_menu_items('Footer Social');
-        $context['footer_primary_menu'] = wp_get_nav_menu_items('Footer Primary');
-        $context['footer_secondary_menu'] = wp_get_nav_menu_items('Footer Secondary');
+
+        if (has_nav_menu('footer-social-menu')) {
+            $footer_social_menu = new TimberMenu('footer-social-menu');
+            $context['footer_social_menu'] = $footer_social_menu->get_items();
+        }
+        else {
+            $context['footer_primary_menu'] = wp_get_nav_menu_items('Footer Social');
```

Comment:
That should be `'footer_social_menu'`, right?

Reply:
Yes! Thanks for spotting it.